conf/messages.cy

```diff
@@ -1173,6 +1209,11 @@ authorisedOfficialPreviousAddress.confirmPage.changeLinkText = Newid cyfeiriad c
 
 authorisedOfficialPreviousAddress.checkYourAnswersLabel = Cyfeiriad cartref blaenorol
 
+# Amend Authorised Official Previous Address Messages
+# ----------------------------------------------------------
+amendAuthorisedOfficialPreviousAddress.title = Diwygio cyfeiriad cartref blaenorol y swyddog awdurdodedig - Swyddogion ac enwebeion
```

Comment:
should `- Swyddogion ac enwebeion` be there?

Reply:
yes.. section name should be part of the title.. so I added that